### PR TITLE
Tutorial main menu

### PR DESCRIPTION
**Pull request by SpeedyRacoon**
Pause button and menu

### CODE REVIEW

File: VR Maze/Assets/Scripts/MoveUp.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	// Use this for initialization
+	void Start () {
+		
+	}
+	
+	// Update is called once per frame
+	void Update () {
+		
+	}
+	public void MoveBoxUp(){
```
Comment (JStuve) [inline VR Maze/Assets/Scripts/MoveUp.cs]: Didn't notice that MoveBoxUp() is being called anywhere. Does MoveUp.cs do anything?

File: VR Maze/Assets/NewBehaviourScript.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NewBehaviourScript : MonoBehaviour {
```
Comment (JStuve) [inline VR Maze/Assets/NewBehaviourScript.cs]: Seems to be added without actually edited. Should be removed to avoid clutter.

Comment (alexweav) [inline VR Maze/Assets/NewBehaviourScript.cs]: This is just a default script which does nothing, can you please remove this file and its .meta file?

**Review by alexweav — CHANGES_REQUESTED** [review]
There's a broken floor tile in the scene, if you load it up you'll notice the floor at the end of the hallway is lopsided. Small issue, but can you fix that?

Code and menu looks good though!

**Review by alexweav — APPROVED** [review]

**Review by JStuve — APPROVED** [review]

**Pull request merged**